gcp/cloudbuild.yaml

```diff
@@ -4,11 +4,12 @@ steps:
   args: ['install']
 - name: node
   entrypoint: npm
-  args: ["run", "create-env"]
+  args: ["run", "create-env-dev"]
   env:
     - 'MAPS_KEY=${_MAPS_PROD_KEY}'
     - 'PASSPORT_CLIENT_ID=${_PASSPORT_PROD_CLIENT_ID}'
     - 'PASSPORT_CLIENT_SECRET=${_PASSPORT_PROD_CLIENT_SECRET}'
+    - 'BASE_URL=${_BASE_URL}'
```

Comment:
Creating separate keys for dev and prod looks great! Currently we are using `_PASSPORT_PROD_CLIENT_ID ` and `_PASSPORT_PROD_CLIENT_SECRET` for testing. Could you apply a set of key for dev purpose?